lib/ProductOpener/Config_off.pm

```diff
@@ -387,16 +389,29 @@ fr:levures
 @taxonomy_fields = qw(states countries languages labels categories additives additives_classes vitamins minerals amino_acids nucleotides other_nutritional_substances allergens traces nutrient_levels misc ingredients nova_groups);
 
 
-# fields in product edit form
+# fields in product edit form, above ingredients and nutrition facts
 
 @product_fields = qw(quantity packaging brands categories labels origins manufacturing_places emb_codes link expiration_date purchase_places stores countries  );
 
+# fields currently not shown in the default edit form, can be used in imports or advanced edit forms
+
+@product_other_fields = qw(
```

Comment:
It could be advanced fields, but it could also be fields that are not going to be exposed in an edit form, like for instance the Nutriscore computed by the producer, or the producer product version id. We get it through imports, we might display it (but not necessarily, it could just be used to compare with the Nutriscore we compute). That's why I put other which is indeed very generic.